Sources/AppStoreConnectCLI/Services/Operations/ListBuildsOperation.swift

```diff
@@ -0,0 +1,61 @@
+// Copyright 2020 Itty Bitty Apps Pty Ltd
+
+import AppStoreConnect_Swift_SDK
+import Combine
+import Foundation
+
+struct ListBuildsOperation: APIOperation {
+
+    struct Options {
+        let filterAppIds: [String]
+        let filterExpired: [String]
+        let filterPreReleaseVersions: [String]
+        let filterBuildNumbers: [String]
+        let filterProcessingStates: [ListBuilds.Filter.ProcessingState]
+        let filterBetaReviewStates: [String]
+        let limit: Int?
+    }
+
+    typealias Build = AppStoreConnect_Swift_SDK.Build
+    typealias Relationships = [AppStoreConnect_Swift_SDK.BuildRelationship]?
+
+    typealias Output  = [(build: Build, relationships: Relationships)]
+
+    private let options: Options
+
+    init(options: Options) {
+        self.options = options
+    }
+
+    func execute(with requestor: EndpointRequestor) -> AnyPublisher<Output, Error> {
+        var filters = [ListBuilds.Filter]()
+        filters += options.filterAppIds.isEmpty ? [] : [ListBuilds.Filter.app(options.filterAppIds)]
+        filters += options.filterPreReleaseVersions.isEmpty ? [] : [ListBuilds.Filter.preReleaseVersionVersion(options.filterPreReleaseVersions)]
+        filters += options.filterBuildNumbers.isEmpty ? [] : [ListBuilds.Filter.version(options.filterBuildNumbers)]
+        filters += options.filterExpired.isEmpty ? [] : [ListBuilds.Filter.expired(options.filterExpired)]
+        filters += options.filterProcessingStates.isEmpty ? [] : [ListBuilds.Filter.processingState(options.filterProcessingStates)]
+        filters += options.filterBetaReviewStates.isEmpty ? [] :  [ListBuilds.Filter.betaAppReviewSubmissionBetaReviewState(options.filterBetaReviewStates)]
+
+        var limit: [ListBuilds.Limit]?
+
+        if let optionLimit = options.limit {
+            limit = [ListBuilds.Limit.individualTesters(optionLimit),
+                     ListBuilds.Limit.betaBuildLocalizations(optionLimit)]
+        }
```

Comment:
This could be simplified to a map if you want:
```suggestion
        let limit = options.limit.map { limit -> [ListBuilds.Limit] in
            [.individualTesters(limit), .betaBuildLocalizations(limit)]
        }
```